### ISSUE TITLE
Binary won't run

### ISSUE THREAD

**Issue by BGMcoder** [dragan224/battle_cats_en_combos#1]
I'm trying to run the BCEN_combos.jar in my Windows7 x64 (running latest Java) and it says "Archive is damaged. Open failed k- Native error: 00048"  - I've downloaded it several times and keep having the same issue.  Is there another jar I can dowload?  I've tried with javaw.exe both 64bit and 32bit.  Thanks

**Comment by dragan224** [comment]
Hey,

Make sure you are using java to open. Right click -> open with java.

I tried to compile the .jar again on windows. Here's the link:
https://www.dropbox.com/s/19kul7mk330dfl2/BCEN_combos_win.jar?dl=0

Can you please let me know if it works? So I can update the thread.

Best,
Dragan

**Comment by BGMcoder** [comment]
Yes, I have done that using the command in the context menu.  I have also just dragged the jar files onto the javaw.exe files to open them.  
Ah - I think I know the problem now - my download manager was nerfing the file.  I was able to download it and run it just fine by bypassing the download manager.  I guess it doesn't like dropbox. 
Thank you for your time and for answering!  I'm sorry to be a bother!
